packages/python/plotly/plotly/express/_core.py

```diff
@@ -1343,7 +1343,8 @@ def init_figure(args, subplot_type, frame_list, nrows, ncols, col_labels, row_la
                 main_size = 0.84
 
             column_widths = [main_size] * (ncols - 1) + [1 - main_size]
-            horizontal_spacing = 0.005
+            width_adjustment = 1 if args["width"] is None else 600 / args["width"]
```

Comment:
For readability / ease of code comprehension, I'd love to have:
* A comment for both of these explaining why these constants are being used here and why we're adjusting for width/height
* More consistency between the width and the height adjustments. It wasn't immediately clear to me that they were doing the same thing